official/resnet/keras/keras_common.py

```diff
@@ -47,15 +55,21 @@ def __init__(self, batch_size):
     self._batch_size = batch_size
     super(TimeHistory, self).__init__()
     self.log_steps = 100
+    self.train_timestamp_history = []
 
   def on_train_begin(self, logs=None):
     self.record_batch = True
 
+  def on_train_end(self, logs=None):
+    self.train_finish_time = time.time()
```

Comment:
Comment (not action needed):  This is likely not totally accurate as I suspect this hook is not called until the graph is compiled and grappler and even XLA run the first time.  I could be wrong.  Not a big deal and it is nice to have.